magento/adobe-stock-integration
# Fix Escape doesn't close the "Couldn't log you in..." message. Instead the Adobe Stock window is closed

### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1542: Escape doesn't close the "Couldn't log you in..." message. Instead the Adobe Stock window is closed

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**, click **Search Adobe Stock**
2. Click **Sign In**
_Couldn't log you in. Login failed. Please check if the Secret Key is set correctly and try again._ message appears
3. Hit Escape button from keyboard

The _Couldn't log you in..._ message is closed
